src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7511,4 +7523,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_UnsupportedPrimaryConstructorParameterCapturingRefAny" xml:space="preserve">
     <value>Cannot use primary constructor parameter of type '{0}' inside an instance member</value>
   </data>
+  <data name="ERR_BadUsingStaticType" xml:space="preserve">
+    <value>'{0}' type is not valid for 'using static'. Only a class, struct, interface, enum, delegate or namespace can be used.</value>
```

Comment:
```suggestion
    <value>'{0}' type is not valid for 'using static'. Only a class, struct, interface, enum, delegate, or namespace can be used.</value>
```

Reply:
- Oxford comma is commonly used, I think. e.g: https://github.com/dotnet/roslyn/blob/591154398bcbcd5793908d62156f7f2366b23426/src/Compilers/CSharp/Portable/CSharpResources.resx#L712
- *Most* errors don't end with period.

```suggestion
    <value>'{0}' type is not valid for 'using static'. Only a class, struct, interface, enum, delegate or namespace can be used</value>
```